pkg/directorysync/client.go

```diff
@@ -137,22 +137,22 @@ type User struct {
 // ListUsersOpts contains the options to request provisioned Directory Users.
 type ListUsersOpts struct {
 	// Directory unique identifier.
-	Directory string
+	Directory string `url:"directory", omitempty`
 
 	// Directory Group unique identifier.
-	Group string
+	Group string `url:"group", omitempty`
 
 	// Maximum number of records to return.
-	Limit int
+	Limit int `url:"limit"`
 
 	// The order in which to paginate records.
-	Order Order
+	Order Order `url:"order,omitempty"`
 
-	// Pagination cursor to receive records before a provided Directory ID.
-	Before string
+	// Pagination cursor to receive records before a provided Event ID.
+	Before string `url:"before,omitempty"`
 
-	// Pagination cursor to receive records after a provided Directory ID.
-	After string
+	// Pagination cursor to receive records after a provided Event ID.
```

Comment:
is the change of `Directory` to `Event` intended here? (it's in a few places but just gonna comment here)